KarhooUISDKTests/KarhooTestConfiguration.swift

```diff
@@ -47,4 +46,8 @@ class KarhooTestConfiguration: KarhooUISDKConfiguration {
     func analytics() -> Analytics {
         mockAnalytics
     }
+    
+    func requireSDKAuthentication(callback: @escaping () -> Void) {
+        //TODO: test this
```

Comment:
It's a really good idea to avoid TODOs... is there a ticket for this instead that can be added to the product backlog?